README.md

```diff
@@ -33,13 +33,26 @@ It's listed on [Elixir - Tools and Data Services Registry](https://bio.tools/nf-
 
 By default, the pipeline currently performs the following:
 
-- Sequencing quality control (`FastQC`)
-- Map Reads to Reference (`BWA mem`)
-- Mark Duplicates (`GATK MarkDuplicates`)
-- Base (Quality Score) Recalibration (`GATK BaseRecalibrator`, `GATK ApplyBQSR`)
-- Preprocessing quality control (`samtools stats`)
-- Preprocessing quality control (`mosdepth`)
-- Overall pipeline run summaries (`MultiQC`)
+- Form consensus reads from UMI sequences (`fgbio`)
+- Sequencing quality control and trimming (`FastQC`, `fastp`)
+- Map Reads to Reference (`BWA-mem` or `BWA-mem2` or `dragmap`)
+- Process BAM file (`GATK MarkDuplicates`, `GATK BaseRecalibrator`, `GATK ApplyBQSR`)
+- Summarise alignment statistics (`samtools stats`, `mosdepth`)
+- Variant calling (enabled by `--tools`, see [compatibility](https://github.com/nf-core/sarek/blob/master/docs/usage.md#which-variant-calling-tool-is-implemented-for-which-data-type)):
+  - `haplotypecaller`
+  - `freebayes`
+  - `mpileup`
+  - `strelka2`
+  - `deepvariant`
+  - `mutect2`
+  - `manta`
+  - `tiddit`
+  - `ASCAT`
+  - `Control-FREEC`
+  - `CNVkit`
+  - `MSIsensor-pro`
+- Variant filtering and annotation (`GATK CNNScoreVariants`, `SnpEff`, `Ensembl VEP`)
```

Comment:
CNNSCore variants is only run for haplotypecaller. In a similar line FiltermutectCalls is run for mutect2 calls

Reply:
In that case maybe we shouldn't add them here? I think as a user it would confuse me more that they are run for 1 out of 5 tools and I can't explcitely enable them, than leaving it out here

Reply:
yes I agree with that. Otherwise I would interpret it as it is avaialble for everything.